### PR TITLE
Fix alignment for Istio logo

### PR DESCRIPTION
**Pull request by shamsher31**
Follow up for https://github.com/istio/istio/pull/27270

See preview: https://github.com/shamsher31/istio/tree/readme-logo

### CODE REVIEW

**Comment by linsun** [comment]
I think i like the current alignment better :)

**Comment by shamsher31** [comment]
> I think i like the current alignment better :)

The current alignment looks off to me. Not sure what is the standard but I can see other projects keep it left aligned.

https://github.com/kubernetes/kubernetes
https://github.com/docker/compose
https://github.com/gohugoio/hugo
https://github.com/kubernetes/kops
https://github.com/moby/moby

**Review by howardjohn — COMMENTED** [review]
I prefer the current as well

**Review by frankbu — COMMENTED** [review]
This is an improvement, but the icon seems much too big. Can we make it about 1/4 of it's current size?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
-<a href="https://istio.io/">
-    <img src="logo/istio-bluelogo-whitebackground-unframed.svg" alt="Istio logo" title="Istio" align="right" width="100" />
-</a>
-
 # Istio
```
Comment (frankbu) [inline README.md:1]: This section heading should come after the go report, godoc, and icon (at line 10).

Comment (shamsher31) [reply README.md]: Moved.

**Comment by shamsher31** [comment]
> This is an improvement, but the icon seems much too big. Can we make it about 1/4 of it's current size?

Setting height and width to 100, see if this looks good? https://github.com/shamsher31/istio/tree/readme-logo

File: README.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
+
 <a href="https://istio.io/">
-    <img src="logo/istio-bluelogo-whitebackground-unframed.svg" alt="Istio logo" title="Istio" align="right" width="100" />
+    <img src="https://github.com/istio/istio/raw/master/logo/istio-bluelogo-whitebackground-unframed.svg"
+         alt="Istio logo" title="Istio" height="100" width="100" />
```
Comment (frankbu) [inline README.md:4]: @shamsher31 The smaller size looks much better, but the arrangement still looks off. Can you try ordering the top items just like K8s does? 1. `# Istio` heading, 2. Go report and go doc, 3. Icon.

Comment (shamsher31) [reply README.md]: Done, now it looks better.

**Review by frankbu — APPROVED** [review]
LGTM

**Review by ericvn — APPROVED** [review]

**Comment by shamsher31** [comment]
@linsun PTAL.

**Comment by shamsher31** [comment]
cc: @howardjohn @linsun

**Review by linsun — APPROVED** [review]
LGTM - thanks!

**Pull request merged**